spec/unit/lib/temporal/client_spec.rb

```diff
@@ -1102,4 +1102,28 @@ class NamespacedWorkflow < Temporal::Workflow
       end
     end
   end
+
+  describe '#count_workflow_executions' do
+    let(:response) do
+      Temporalio::Api::WorkflowService::V1::CountWorkflowExecutionsResponse.new(
+        count: 5
+      )
+    end
+
+    before do
+      allow(connection)
+        .to receive(:count_workflow_executions)
+        .and_return(response)
+    end
+
+    it 'returns the count' do
+      resp = subject.count_workflow_executions(namespace, '')
```

Comment:
nit: having a non-empty query makes the test slightly more realistic (at least I don't think an empty query works on the real server)

Reply:
Believe it or not, an empty query _does_ work on the real server (see slack), but I've gone ahead and modified the test to be more realistic (there's very few scenarios during which you'd do an empty query).